beetsplug/fetchart.py

```diff
@@ -1401,7 +1526,8 @@ def batch_fetch_art(self, lib, albums, force, quiet):
                 # In ordinary invocations, look for images on the
                 # filesystem. When forcing, however, always go to the Web
                 # sources.
-                local_paths = None if force else [album.path]
+                # FIXME: Avoid the cast by somehow typing beets.library.Item
```

Comment:
Yeah, I guess the solution is to special-case the `path` field, because it is so commonly used. Otherwise, `Item` and `Album` behave pretty much like dictionaries, which can store arbitrary values. How to deal with this for other fields is one of the biggest issues that we still need to solve when typing all of beets.